composer.json

```diff
@@ -11,10 +11,10 @@
     ],
     "require": {
         "php": "^7",
-        "symfony/symfony": "^2.7",
+        "symfony/symfony": "^3.4",
         "doctrine/doctrine-bundle": "^1",
         "sonata-project/doctrine-orm-admin-bundle": "^3 || ^2",
-        "zicht/framework-extra-bundle": "^4.1 || ^5",
+        "zicht/framework-extra-bundle": "4 - 6",
```

Comment:
should be >= 6, others wont support php7, perhaps wise to check other dependencies?